src/config/userTypes.ts

```diff
@@ -32,33 +32,43 @@ export interface IHacker {
   id: string;
   accountId: string | IAccount; // for querying account as well
   status: HackerStatus;
-  school: string;
-  degree: string;
-  // no enum for these
-  gender?: string;
-  needsBus?: boolean;
   application: {
-    portfolioURL: {
-      // gcloud bucket link
-      resume: string;
-      github?: string;
-      dropler?: string;
-      personal?: string;
-      linkedIn?: string;
-      other?: string;
```

Comment:
This schema follows the mongo schema in the database, and cannot be changed on a whim like this unfortunately

Reply:
https://github.com/hackmcgill/hackerAPI/pull/603